client/src/store/historyStore/historyStore.js

```diff
@@ -84,20 +84,20 @@ const promises = {
 const actions = {
     async copyHistory({ dispatch }, { history, name, copyAll }) {
         const newHistory = await cloneHistory(history, name, copyAll);
-        await dispatch("selectHistory", newHistory);
+        return dispatch("selectHistory", newHistory);
```

Comment:
I would like to keep it consistent here, there is also no need for the `copyHistory` function to `await` the result of the last call, since the `await` is not followed by any operation.

Reply:
Adding to this conversation, I found this document here titled `Best Practices for ES2017 Asynchronous Functions (`async`/`await`)` available at https://dev.to/somedood/best-practices-for-es2017-asynchronous-functions-async-await-39ji.
In the section titled `Refrain from using return await` the author states: `Generally speaking, the final promise inside an async function should be returned directly.`. This is followed by a section titled: `Prefer simple promises instead` in which it is stated that: `This has led many of us to write async functions by default, even when a simple promise (without the async wrapper) would suffice.` followed by `If an async function only serves to wrap a single promise or two, perhaps it is more optimal to forego the async wrapper altogether.`.